Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -559,15 +557,31 @@ UpdateFspConfig (
         break;
 
       case ProbeTypeManual:
-        Fspmcfg->DciModphyPg              = 0x1;
-        Fspmcfg->DciDbcMode               = 0x0;
-        Fspmcfg->CpuTraceHubMode          = 0x0;
-        Fspmcfg->PchTraceHubMode          = 0x0;
+        Fspmcfg->DciEn                    = MemCfgData->DciEn;
+        Fspmcfg->DciModphyPg              = MemCfgData->DciModphyPg;
+        Fspmcfg->DciUsb3TypecUfpDbg       = MemCfgData->DciUsb3TypecUfpDbg;
+        Fspmcfg->DciDbcMode               = MemCfgData->DciDbcMode;
+        Fspmcfg->CpuTraceHubMode          = MemCfgData->CpuTraceHubMode;
+        Fspmcfg->PchTraceHubMode          = MemCfgData->PchTraceHubMode;
+        Fspmcfg->DebugInterfaceEnable     = 0x0;

```

Comment:
why do we disable DebugInterface ?  Should it be configurable ?

Reply:
currently this YAML is missing. Will set it to configurable in part 2.

Reply:
got it